setup.cfg

```diff
@@ -48,3 +48,6 @@ parallel = true
 
 [coverage:report]
 show_missing = true
+
+[isort]
+multi_line_output = 5
```

Comment:
Why do you want this addition? Can it be made in a separate PR?

Reply:
That was accidental, sorry! I'll remove it.